src/Raven.Client/Raven.Client.csproj

```diff
@@ -3,7 +3,7 @@
     <Version></Version>
     <Description>RavenDB Client is the client library for accessing RavenDB</Description>
     <Authors>Hibernating Rhinos</Authors>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.1;netstandard2.0</TargetFrameworks>
```

Comment:
Remember to include the Sparrow.dll in NuGet for netcoreapp2.1. In v4.1 you can use the netcoreapp2.1 Sparrow.dll. In v4.0 you need to use the netstandard2.0 one.